### PR TITLE
backticks DocType Element to standard and rephrases "you"

### PR DESCRIPTION
**Pull request by ablwr**
Just two small adjustments.

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -416,7 +416,7 @@ The `recursive` attribute is OPTIONAL. If the `recursive` attribute is not prese
 
 The `minver` (minimum version) attribute stores a non-negative integer that represents the first version of the `docType` to support the `EBML Element`.
 
-The `minver` attribute is OPTIONAL. If the `minver` attribute is not present then the `EBML Element` has a minimum version of "1". 
+The `minver` attribute is OPTIONAL. If the `minver` attribute is not present then the `EBML Element` has a minimum version of "1".
```
Comment (mbunkus) [inline specification.markdown]: Weeeeeell… if you're touching that line, then you might as well add a comma between "present" and "then".

I've found something to complain! Yay! :grin:

Comment (dericed) [inline specification.markdown]: #notblocking ;)

Comment (ablwr) [inline specification.markdown]: OK. Have a few more small changes coming down the pipeline but I'll adjust this one here, squash, and submit again.

Comment (ablwr) [inline specification.markdown]: And ready.

**Comment by mbunkus** [comment]
Great, thanks 😄

**Pull request merged**